src/dex/wombat/wombat.ts

```diff
@@ -0,0 +1,436 @@
+import { AsyncOrSync, DeepReadonly } from 'ts-essentials';
+import { Interface } from '@ethersproject/abi';
+import { SwapSide } from '@paraswap/core';
+
+import {
+  AdapterExchangeParam,
+  Address,
+  ExchangePrices,
+  Logger,
+  PoolLiquidity,
+  PoolPrices,
+  SimpleExchangeParam,
+  Token,
+} from '../../types';
+import { Network } from '../../constants';
+import * as CALLDATA_GAS_COST from '../../calldata-gas-cost';
+import { getBigIntPow, getDexKeysWithNetwork, isETHAddress } from '../../utils';
+import { IDex } from '../idex';
+import { IDexHelper } from '../../dex-helper';
+import { DexParams, PoolState, WombatData } from './types';
+import {
+  getLocalDeadlineAsFriendlyPlaceholder,
+  SimpleExchange,
+} from '../simple-exchange';
+import { Adapters, WombatConfig } from './config';
+import PoolABI from '../../abi/wombat/pool.json';
+import AssetABI from '../../abi/wombat/asset.json';
+import ERC20ABI from '../../abi/erc20.json';
+import { WombatQuoter } from './wombat-quoter';
+import { WombatBmw } from './wombat-bmw';
+import { fromWad } from './utils';
+import { WombatPool } from './wombat-pool';
+import { StatePollingManager } from '../../lib/stateful-rpc-poller/state-polling-manager';
+
+export class Wombat extends SimpleExchange implements IDex<WombatData> {
+  // export class Wombat implements IDex<WombatData> {
```

Comment:
@KanievskyiDanylo Could you please remove this comment? I think it doesn't make too much sense.